### PR TITLE
Fixing broken compatibility

### PR DESCRIPTION
**Pull request by timorl**
# Description

Compatibility didn't work, because apparently no one checked?

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

### CODE REVIEW

File: finality-aleph/src/party/manager/mod.rs [hunk 1]
```diff
@@ -348,11 +348,13 @@ where
                 info!(target: "aleph-party", "Running session with legacy-only AlephBFT version.");
                 self.legacy_subtasks(params)
             }
-            Ok(version) if version == CURRENT_VERSION => {
+            // The `as`es here should be removed, but this would require a pallet migration and I
```
Comment (maciejnems) [inline finality-aleph/src/party/manager/mod.rs:351]: <3

**Review by maciejnems — APPROVED** [review]
ups

**Review by mike1729 — APPROVED** [review]
D'oh!

File: scripts/run_nodes.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ validator_ids_string="${validator_ids[*]}"
 validator_ids_string="${validator_ids_string//${IFS:0:1}/,}"
 
 echo "Bootstrapping chain for nodes 0..$((N_VALIDATORS - 1))"
-./target/release/aleph-node bootstrap-chain --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
+./target/release/aleph-node bootstrap-chain --raw --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
```
Comment (Marcin-Radecki) [inline scripts/run_nodes.sh:55]: Why we added `--raw` here? This script is used mostly for development purposes to my knowledge, and having non-raw chainspec makes debugging easier when it comes to e.g. adding new pallet

Comment (timorl) [reply scripts/run_nodes.sh:55]: But not having a raw chainspec also produces cryptic errors when you are trying to run it on two different versions.

Perhaps we could produce both?

**Pull request merged**